src/SFA.DAS.EmployerAccounts/Services/DasRecruitService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Diagnostics;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using SFA.DAS.EAS.Portal.Client.Http;
+using SFA.DAS.EmployerAccounts.Features;
+using SFA.DAS.EmployerAccounts.Models.Recruit;
+using SFA.DAS.NLog.Logger;
+
+namespace SFA.DAS.EmployerAccounts.Services
+{
+    public class DasRecruitService : IDasRecruitService
```

Comment:
Does this class have unit tests?

Reply:
There are full fledged test cases already available in the EAS.Portal solution, done by Phil. Hence did not write tests for this and its similar to the one used in EAS.Portal. If required, i will add some more tests.

Reply:
So has this class been copy & pasted from EAS.Portal?

Reply:
This has now beem covered by tests.